core/init.js

```diff
@@ -0,0 +1,33 @@
+/*
+   init.js is appending to our minified bundles in the CP build process.
```

Comment:
It is appended by our build scripts. ```init.js``` is supposed to kick everything off when we are minified.